packages/validator/src/slashingProtection/minMaxSurround/minMaxSurround.ts

```diff
@@ -5,13 +5,29 @@ import {SurroundAttestationError, SurroundAttestationErrorCode} from "./errors.j
 // surround vote checking with min-max surround
 // https://github.com/protolambda/eth2-surround#min-max-surround
 
+/**
+ * Number of epochs in the past to check for surrounding attestations.
+ *
+ * This value can be limited to a reasonable high amount as Lodestar does not solely rely on this strategy but also
+ * implements the minimal strategy which has been formally proven to be safe (https://github.com/michaelsproul/slashing-proofs).
```

Comment:
Minimal strategy refers to [conditions 4 and 5](https://eips.ethereum.org/EIPS/eip-3076#conditions) from EIP-3076 and is implemented in Lodestar

https://github.com/ChainSafe/lodestar/blob/3ed8d04c7971f753570f0d054a3c44bc2c382be1/packages/validator/src/slashingProtection/attestation/index.ts#L99-L121